user_research/Device/digitalData.md

```diff
@@ -0,0 +1,6 @@
+# 2012 AEC![devices](../Pictures/Devices.png)
+![ageInternet](../Pictures/ageAppropriation.png)
+![ageSmartphone](../Pictures/smartphoneAge.png)
+![mobinautes](../Pictures/mobinautes.png)
+# 2011 CREDOC
```

Comment:
[CREDOC](http://www.credoc.fr/)  is the "Centre de Recherche pour l'étude et l'observation des conditions de vie".

I will updates this file with links and a little comment about the data.

CREDOC data come from a study titled "ÉVOLUTIONS DU COMPORTEMENT DES FRANÇAIS FACE AU DEVELOPPEMENT DE L’ECONOMIE CIRCULAIRE".

So the gap between data can be also explained by the difference between population : Credoc observed french people in general whereas AEC observed people living in Aquitaine